env.sh.example

```diff
@@ -21,8 +21,8 @@ export VERTEX_LOCATION=europe-west2
 export VERTEX_NETWORK= # optional
 export VERTEX_PROJECT_ID=my-gcp-project
 
-# Suffix (e.g. '_<your name>') to facilitate running concurrent pipelines in the same Google Cloud project. Populate if working in a team to avoid overwriting resources during development 
-export RESOURCE_SUFFIX=
+# Suffix (e.g. '_<your name>') to facilitate running concurrent pipelines in the same Google Cloud project. Change if working in a team to avoid overwriting resources during development 
+export RESOURCE_SUFFIX=_default
```

Comment:
I wonder if it should be "default" instead of "_default" - leave the underscore out of the suffix and instead put it in the table names since it only really makes sense for BQ tables

Reply:
Yeah good idea, now that the resource_suffix is always going to be populated, we can handle it easily in the `pipeline.py`